lib/index.js

```diff
@@ -5,6 +5,19 @@ $(document).ready(() => {
   foodsRequests.getFoods()
   $('#submit-food').on('click', event => {
     event.preventDefault()
-    foodsRequests.addNewFood()
+    newFoodSequence()
   })
 })
+
+const newFoodSequence = () => {
+  var nameInput = $('#food-name').val()
+  if (nameInput != '') {
```

Comment:
Since this if else is straightforward, we could make it into a one line method to make it cleaner visually. 

nameInput !== '' ? foodsRequests.addNewFood() : validateNameAlert()

This uses the ternary operator. Here's a link to documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator